build-system/tasks/extension-generator/index.js

```diff
@@ -313,7 +313,7 @@ async function makeAmpExtension() {
   const filenames = Object.keys(fileContent)
     // Don't format .html because AMP boilerplate would expand into multiple lines.
     .filter((filename) => !filename.endsWith('.html'));
-  execOrThrow(`npx prettier --ignore-unknown --write ${filenames.join(' ')}`);
+  execOrThrow(`npx prettier --ignore-unknown --write ${filenames.join(' ')}`, '');
```

Comment:
Instead of adding empty strings to every `execOrThrow` call, can you change the signature of that function to have a default value of `''` for the second arg?

Reply:
There are only a couple places where the second argument is missing and enforcing better error messages seems like a good thing.